ddtrace/internal/module.py

```diff
@@ -428,7 +428,7 @@ def after_import(self, module):
             for hook in hooks:
                 hook(module)
         if config._telemetry_enabled and config._telemetry_dependency_collection:
```

Comment:
This is still a potential source of circular import. We should not have any form of "product"-specific details in this general utility module

Reply:
Is the "config"module product-specific? Anyway I can move the checks to the writer.

Reply:
@P403n1x87 change done, PTAL.

Reply:
@P403n1x87 change done, PTAL.

Reply:
Hi @P403n1x87 can you un-request the change since it's done?